SampleOffer.sol

```diff
@@ -53,35 +53,4 @@ contract SampleOffer is SampleOfferWithoutReward {
     function setDeploymentReward(uint _deploymentReward) onlyClient noEther {
```

Comment:
No that would not really make sense since that's for a contract that should have no rewards, and this is the rewards extension.